src/docs/components/DocsActiveBlock.jsx

```diff
@@ -28,7 +28,8 @@ class DocsActiveBlock extends Component {
     this.state = {
       showCode: null,
       changeBackground: 'light',
-      props: componentProps
+      props: componentProps,
+      renderNormally: true
```

Comment:
hack to force recreation of component every time settings change (instead of just updating it's props)